04LessonFour/in-class-project/index.html

```diff
@@ -2,72 +2,89 @@
 
 <html>
 
-<head>
+  <head>
 
-    <title>The Daily Planet</title>
-    <link rel="stylesheet" href="css/style.css">
+    <title>Intel &reg;</title>
 
-</head>
+    <link rel="stylesheet" href="css/style.css">
 
-<body>
+  </head>
 
+  <body>
+    <!--Still curious about fonts. These do not match the previous page, but replicating it was not as simple as imagined. Change later when more fonts are installed.  -->
     <header>
 
-        <h1>Intel</h1>
-        <nav>
-            <a href="www.intel.com">Intel</a>
-            <a href="www.intel.com"> Processor</a>
-        </nav>
+      <h1>Intel&reg; </h1>
+
+      <nav>
+
+        <a href="http://www.intel.com/content/www/us/en/processors/core/core-i7-processor.html" alt="Intel Core i7 Procesor">i7 Processor</a>
+        <a href="http://www.intel.com/content/www/us/en/solid-state-drives/solid-state-drives-ssd.html" alt="Solid State Drives">Solid State Drives</a>
+
+      </nav>
 
     </header>
+    <!--Main is a container for formatting and style  -->
+    <main>
+
+
+      <!--Each section has an id for organizational purposes  -->
+      <section id="processor">
+
+        <h2>Intel &reg; Core&trade; i7</h2>
+
+        <p>The 6th Generation Intel &reg; Core&trade; i7 processors deliver a new class of computing with a host of new features to power the next generation of desktops, laptops, and 2 in 1 PCs. Expect lighting fast speeds and peak performance through even
+          the toughest of tasks and games.</p>
+
+      </section>
 
-    <section id="sec_processor">
+      <section id="ready">
 
-        <h2>Intel® Core™ i7</h2>
-        <p>The 6th Generation Intel® Core™ i7 processors deliver a new class of computing with a host of new features to power the next generation of desktops, laptops, and 2 in 1 PCs. Expect lighting fast speeds and peak performance through even the toughest of tasks and games.</p>
-        <table>
-    </section>
+        <img id="proc" src="img/processor.jpg" alt "processor" />
+        <!--Is it acceptable to use the br tag to create space? It seems easier than padding but will change if it isn't the best coding practice. -->
+        <br />
```

Comment:
instead of a br, try using margin. creating space between elements is typically done this way. line breaks are more for paragraphs of text.